### PR TITLE
[PEER-1047] Notifications for Sui blockchain

### PR DESCRIPTION
**Pull request by IgorAlexeew**
PR for [PEER-1047](https://peer.youtrack.cloud/issue/PEER-1047)

### CODE REVIEW

**Review by sergeyi23 — APPROVED** [review]
Looks good.

File: src/core/sui-blockchain/sui.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export async function queryEvents(
 }
 
 export async function getObject(objectId: string) {
-  const endpoint = await getSecretValue('SUI_RPC_ENDPOINT');
+  const endpoint = process.env.SUI_RPC_ENDPOINT;
```
Comment (andreilarin) [inline src/core/sui-blockchain/sui.ts]: do the same with all calls `getSecretValue` in this file

**Review by andreilarin — CHANGES_REQUESTED** [review]

**Review by andreilarin — APPROVED** [review]

File: src/core/sui-blockchain/sui.ts [hunk 2]
```diff
@@ -19,9 +18,9 @@ export async function queryEvents(
   cursor: string | undefined | null,
   maxTxNumber: number
 ) {
-  const endpoint = await getSecretValue('RPC_SUI_ENDPOINT');
+  const endpoint = process.env.SUI_RPC_ENDPOINT;
```
Comment (sergeyi23) [inline src/core/sui-blockchain/sui.ts]: Rollback to secret value

File: src/core/sui-blockchain/sui.ts [hunk 3]
```diff
@@ -1,11 +1,10 @@
 import { JsonRpcProvider, Connection } from '@mysten/sui.js';
 import { ConfigurationError } from 'src/core/errors';
-import { getSecretValue } from 'src/core/utils/secrets';
 
 export async function createSuiProvider() {
-  const endpoint = await getSecretValue('RPC_SUI_ENDPOINT');
+  const endpoint = process.env.SUI_RPC_ENDPOINT;
```
Comment (sergeyi23) [inline src/core/sui-blockchain/sui.ts]: Rollback to getSecretValue

File: src/core/sui-blockchain/sui.ts [hunk 4]
```diff
@@ -90,9 +89,9 @@ export async function getDynamicFieldObject(
   type: string,
   index: string
 ) {
-  const endpoint = await getSecretValue('RPC_SUI_ENDPOINT');
+  const endpoint = process.env.SUI_RPC_ENDPOINT;
```
Comment (sergeyi23) [inline src/core/sui-blockchain/sui.ts]: rollback to getSecretValue

File: stages/dev2.yml [hunk 5]
```diff
@@ -29,3 +30,7 @@ environment:
   SUI_PACKAGE_ADDRESS: '0x537ae9bb80ec56b152271a7a183a7f5e6cb3db0fff7c6221f2f7f25c8cf0e0ca'
   SUI_USERS_RATING_COLLECTION: '0x656bb8096918da648a306c4a285888e8207aee61f818a4758f86d987f3c8d198'
   RPC_SUI_ENDPOINT: 'https://rpc.ankr.com/sui_testnet'
+
+  SUI_PACKAGE_ADDRESS: '0x537ae9bb80ec56b152271a7a183a7f5e6cb3db0fff7c6221f2f7f25c8cf0e0ca'
```
Comment (sergeyi23) [inline stages/dev2.yml]: Duplicate config values

File: stages/dev3.yml [hunk 6]
```diff
@@ -29,3 +30,7 @@ environment:
   SUI_PACKAGE_ADDRESS: '0x83f909002b471f7e1dbec27c837ff143189f51dff9faf42697542df82113b2ec'
   SUI_USERS_RATING_COLLECTION: '0x16e3c9f176effde60b1e258afbb9ceb6b2b7eecb2152a836c6ab633f16fc5056'
   RPC_SUI_ENDPOINT: 'https://fullnode.testnet.sui.io'
+
+  SUI_PACKAGE_ADDRESS: '0x83f909002b471f7e1dbec27c837ff143189f51dff9faf42697542df82113b2ec'
```
Comment (sergeyi23) [inline stages/dev3.yml]: Duplicate config values

**Review by sergeyi23 — APPROVED** [review]
Fix comments before merge

**Pull request merged**